### PR TITLE
[OTA-3898] Bad request message when switching between two org

### PR DESCRIPTION
**Pull request by piotrekc**
Signed-off-by: Piotr Cichon <ext-piotr.cichon@here.com>

### CODE REVIEW

File: ota-plus-web/app/reactapp/package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "antd": "3.15.2",
     "axios": "0.19.0",
     "chart.js": "2.8.0",
+    "encodeurl": "^1.0.2",
```
Comment (piotrsmol) [inline ota-plus-web/app/reactapp/package.json]: specify exact version

**Comment by houcros** [comment]
Is `package-lock.json` supposed to be part of the commit? The actual fix was a couple of lines but we have a huge diff 😕

File: ota-plus-web/app/reactapp/package.json (hunk 1 above)
Comment (piotrekc) [reply ota-plus-web/app/reactapp/package.json]: Done

**Comment by piotrekc** [comment]
> Is `package-lock.json` supposed to be part of the commit? The actual fix was a couple of lines but we have a huge diff 😕

Yes, https://docs.npmjs.com/files/package-lock.json

**Pull request merged**